app.js

```diff
@@ -0,0 +1,129 @@
+let squareChange = document.getElementsByClassName('squares');
+let table1 = document.querySelector('table');
+let td = document.getElementsByTagName('td');
+let turn = 0;
+let resetButton = document.querySelector('button');
+let winCo = false;
+
+
+
+
+
+
+document.addEventListener('DOMContentLoaded', function() {
+table1.addEventListener('click', event => {
```

Comment:
Sameh - lines 14 through 117 are nested inside of the document.addEventListner("DOMContentLoaded.... so all of those lines would be tabbed over one more time.